projects/ccf-rui/src/app/shared/components/extraction-sites-menu/extraction-sites-menu.component.scss

```diff
@@ -0,0 +1,17 @@
+:host {
+  display: flex;
+  flex-direction: column;
+  margin-left: .5rem !important;
+  margin-top: .5rem !important;
+
+  .site-entry {
+    display: flex;
+
+    .site-name {
+      margin-left: 1rem;
+      font-size: 14px;
```

Comment:
We should be using rem units for font sizes (I realize he has px in the specs but they should still be specified in rems). Further, on [page 3 of the spec](https://drive.google.com/file/d/1xp2SFiCxBCGM4OaT_Dni839p2w53IuT2/view?usp=sharing) it says 
> Almost all text is in 16 point. The other two sizes should be auto-determined by the framework as they are used by elements such as inputs and buttons.

Is there a reason why 14px font is used here?